components/navigation.js

```diff
@@ -201,479 +34,39 @@ NavigationLink.propTypes = {
-  toggleNavCollapsed: PropTypes.func.isRequired,
-};
-
-/**
- * Wraps the navigation items in <nav> and <ul> tags.
- */
-function NavigationList({ className = null, children }) {
-  return (
-    <nav className={className}>
-      <ul className="[&>li]:block">{children}</ul>
-    </nav>
-  );
-}
-
-NavigationList.propTypes = {
-  // Optional Tailwind CSS class name to add to the nav element
-  className: PropTypes.string,
-};
-
-/**
- * Renders the navigation area for mobile and desktop.
- */
-function NavigationExpanded({ navigationClick, toggleNavCollapsed }) {
-  // Holds the ids of the currently open parent navigation items
-  const [openedParents, setOpenedParents] = React.useState([]);
-
-  /**
-   * Called when the user clicks a group navigation item to open or close it.
-   * @param {string} parentId ID of the clicked parent navigation item
-   */
-  function handleParentClick(parentId) {
-    if (openedParents.includes(parentId)) {
-      // Close the parent navigation item.
-      setOpenedParents(openedParents.filter((id) => id !== parentId));
-    } else {
-      // Open the parent navigation item.
-      setOpenedParents([...openedParents, parentId]);
-    }
-  }
-
+export default function Navigation() {
   return (
     <>
-      {toggleNavCollapsed && (
-        <NavigationLogo
-          toggleNavCollapsed={toggleNavCollapsed}
-          isNavCollapsed={false}
-        />
-      )}
-      <NavigationList className="px-7">
-        <NavigationGroupItem
-          id="data"
-          title="Data"
-          icon={<Icon.DataModel />}
-          isGroupOpened={openedParents.includes("data")}
-          handleGroupClick={handleParentClick}
-        >
-          <NavigationHrefItem
-            id="experiments"
-            href="https://www.encodeproject.org/search/?type=Experiment&internal_tags=RegulomeDB_2_2"
-            navigationClick={navigationClick}
-            isChildItem
-          >
-            Experiments
-          </NavigationHrefItem>
-          <NavigationHrefItem
-            id="annotations"
-            href="https://www.encodeproject.org/search/?type=Annotation&internal_tags=RegulomeDB_2_2"
-            navigationClick={navigationClick}
-            isChildItem
-          >
-            Annotations
-          </NavigationHrefItem>
-        </NavigationGroupItem>
-        <NavigationHrefItem
-          id="help"
-          href="/help"
-          navigationClick={navigationClick}
-        >
-          <NavigationIcon>
-            <InformationCircleIcon />
-          </NavigationIcon>
-          Help
-        </NavigationHrefItem>
-        <NavigationHrefItem
+      <div className="flex justify-end">
+        <NavigationLink
           id="contact"
-          navigationClick={navigationClick}
           href="mailto:regulomedb@mailman.stanford.edu"
-          target="_blank"
-          rel="noreferrer"
           aria-label="Email the RegulomeDB help desk"
+          isExternalLink={true}
```

Comment:
In JSX you can also just have `isExternalLink` by itself and it implies a value of true, if that’s more convenient.